models/packages/debian/search.go

```diff
@@ -62,28 +61,40 @@ func SearchLatestPackages(ctx context.Context, opts *PackageSearchOptions) ([]*p
 		})
 	}
 
-	cond = cond.
-		And(builder.Expr("pv2.id IS NULL"))
```

Comment:
Looks like this condition has disappeared?

Reply:
Yes, that's correct. This condition was used to select the latest version.